src/Game.js

```diff
@@ -106,15 +109,15 @@ export default class Game {
             Math.sqrt(serverDiffX ** 2 + serverDiffY ** 2) >
             allowedServerDivergencePx
           ) {
-            playerObj.x = player.x;
-            playerObj.y = player.y;
+            playerObj.x = player.x * blockWidth;
+            playerObj.y = player.y * blockHeight;
```

Comment:
X will be from 0 - 16 so we multiply it by blockWidth to find the actual position